terraform/modules/baseline/schedule_alarms_lambda.tf

```diff
@@ -1,6 +1,11 @@
 module "schedule_alarms_lambda" {
   source = "../schedule_alarms_lambda"
 
+  count = (
+    var.schedule_alarms_lambda.alarm_list == [] ||
```

Comment:
Doesn't this need to be &&, i.e. don't create if both alarm_list and alarm_patterns is null.  Alternatively, just check var.schedule_alarms_lambda == null, and have null as the default value for var.schedule_alarms_lambda

Reply:
It's creating the lambda and all the resources even if the environment doesn't specify a list or patterns. It's either/or between the `alarm_list` and `alarm_patterns` args. So I just need check if at least one of the two arguments has a value.

However, in rubber-duck-style, have made me realise I need to flip around the logic to this:
```hcl
(
  var.schedule_alarms_lambda.alarm_list != [] ||
  var.schedule_alarms_lambda.alarm_patterns != []
) ? 1 : 0
```